src/components/UserLogin.vue

```diff
@@ -18,14 +18,15 @@
       </div>
 
       <div class="form-actions">
-        <input type="submit" value="Login" />
+        <p><input type="submit" value="Login" /> &ensp; Don't have an account yet? <router-link :to="{path: '/registration'}">Register</router-link></p>
```

Comment:
The login button shouldn't be part of the paragraph. We need to split the elements, e.g. something like:
<div class="form-actions">
  <div class="form-action">...</div>
  <div class="form-action">...</div>
</div>